### DISCUSSION TITLE
"4.31. Drawing marks on window decoration": "If activated, marks (see [vim_like_marks]) on windows are drawn in their window decoration", "The default for this option is yes" - but in windows decorations I see only title

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [i3/i3#6043] [Q&A]
Looks like documentation about marks should be improved?

https://i3wm.org/docs/userguide.html#show_marks

**Comment by okraits** [comment]
I don't see any issue with the documentation. The behaviour is as explained:

* `show_marks yes` (default): all marks except marks starting with `_` are shown in the title bar on the right side
* `show_marks no`: no marks are shown
